mlflow/types/chat.py

```diff
@@ -109,15 +120,83 @@ class FunctionToolDefinition(BaseModel):
     strict: Optional[bool] = None
 
 
-class UnityCatalogFunctionToolDefinition(BaseModel):
-    name: str
-
-
 class ChatTool(BaseModel):
-    type: Literal["function", "uc_function"]
+    """A tool definition passed to the chat completion API."""
+
+    type: Literal["function"]
     function: Optional[FunctionToolDefinition] = None
-    uc_function: Optional[UnityCatalogFunctionToolDefinition] = None
 
 
 class ChatTools(BaseModel):
     tools: Optional[list[ChatTool]] = Field(None, min_items=1)
+
+
+class BaseRequestPayload(BaseModel):
+    """Common parameters used for chat completions and completion endpoints."""
+
+    temperature: float = Field(0.0, ge=0, le=2)
+    n: int = Field(1, ge=1)
+    stop: Optional[list[str]] = Field(None, min_items=1)
+    max_tokens: Optional[int] = Field(None, ge=1)
+    stream: Optional[bool] = None
+    model: Optional[str] = None
+
+
+class ChatCompletionRequest(ChatTools, BaseRequestPayload):
```

Comment:
note: moving these classes from `mllfow.gateway.schema` so that LangChain modules do not depend on Gateway.